gcp_variant_transforms/transforms/limit_write_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for limit_write module."""
+
+from __future__ import absolute_import
+
+import unittest
+
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+from apache_beam.transforms import Create
+
+from gcp_variant_transforms.beam_io import vcfio
+from gcp_variant_transforms.transforms import limit_write
+from gcp_variant_transforms.transforms.variant_to_bigquery import _WRITE_SHARDS_LIMIT as DefaultShardLimit
+
+
+class LimitWriteTest(unittest.TestCase):
+  """Test cases for the ``LimitWrite`` PTransform."""
+
+  def _get_sample_variants(self):
+    variant1 = vcfio.Variant(
+        reference_name='chr19', start=11, end=12, reference_bases='C',
+        alternate_bases=['A', 'TT'], names=['rs1', 'rs2'], quality=2,
+        filters=['PASS'],
+        info={'IFR': vcfio.VariantInfo([0.1, 0.2], 'A'),
```

Comment:
Is there a reason to include all of these details? Looks like you should be able to just test with basic variant info (ref_name, start, end, maybe reference_bases) given that you're not looking at the content in the implementation anyway.